src/Components/Inputs/Select.php

```diff
@@ -119,6 +127,20 @@ private function initSrcValues(array $additional): void
             ->all();
     }
 
+    private function initSrcResources(array $additional): void
+    {
+        //add in stuff for valueProperty
+        $resourceList = \App\Http\Controllers\ResourceController::getResourceList();
```

Comment:
The resources should be registered with another class that's part of the package, and the component can initialize form that.

There's no guarantee the client application will keep the stub controller; they may move it to another folder or rename it, so the lib cannot rely on `App\Http\Controllers\ResourceController` existing.